pkg/tools/interfaces.go

```diff
@@ -25,6 +25,8 @@ const (
 	EtcdErrorCodeTestFailed    = 101
 	EtcdErrorCodeNodeExist     = 105
 	EtcdErrorCodeValueRequired = 200
+	EtcdErrorCodeWatchExpired  = 401
+	EtcdErrorCodeUnreachable   = 501
```

Comment:
> NotImplemented (501) over ServiceUnavailable (503)?

Oh, these are just random constant values?  Can you shift the new ones out of the HTTP error code ranges?